terraform/modules/api_gateway/gateway/main.tf

```diff
@@ -48,7 +48,7 @@ module "query_get" {
 
   require_api_key = true
   http_method     = "GET"
-  authorization   = "NONE"
+  authorization   = "CUSTOM"
```

Comment:
Oh, was this something we missed when launching API keys?

Reply:
yup but no big deal.... the ability to add api key to query parameter was added for open data portal that hits the download (and not query endpoints) that already have this functionality enabled...

Reply:
D'oh! Sorry, I forgot this when we moved to production with the API keys.